index.html

```diff
@@ -343,16 +349,16 @@
 
 		Diese kostenlose Open-Source-Anleitung wurde ermöglicht dank
 		Nickys Unterstützer auf Patreon.
-		Wenn sie (und nur dann) in diesen schweren Zeiten über ein ausreichendes Einkommen verfügen, können Sie
+		Wenn du (und nur dann) in diesen schweren Zeiten über ein ausreichendes Einkommen verfügst, kannst du
```

Comment:
```suggestion
		Wenn du in diesen schweren Zeiten über ein mehr als ausreichendes Einkommen verfügst (und wirklich NUR DANN), kannst du
```
näher am Ton des Originals, finde ich hier besonders wichtig da es ja um einen Aufruf für Spenden direkt an Nicky geht